discovery/kubernetes/pod.go

```diff
@@ -141,6 +141,7 @@ const (
 	podContainerPortNameLabel     = metaLabelPrefix + "pod_container_port_name"
 	podContainerPortNumberLabel   = metaLabelPrefix + "pod_container_port_number"
 	podContainerPortProtocolLabel = metaLabelPrefix + "pod_container_port_protocol"
+	podContainerIsInit            = metaLabelPrefix + "pod_container_init"
```

Comment:
Just for the sake of clarity: I thought in this PR the consensus was on `pod_container_is_init` is it a bug or `is` is omitted intentionally? I personally think `pod_container_is_init` makes more sense.
```suggestion
	podContainerIsInit            = metaLabelPrefix + "pod_container_is_init"
```

Reply:
Ignore me, seen https://github.com/prometheus/prometheus/pull/5598#discussion_r288481795 comment.